corehq/apps/ota/views.py

```diff
@@ -37,6 +38,7 @@
 from .utils import demo_user_restore_response, get_restore_user, is_permitted_to_restore, handle_401_response
 
 
+@location_safe
```

Comment:
ah interesting, didn't think about that, but yes that would make sense. is there additional work to making case claim location safe or can i just add the decorator? (if more work i might punt to a follow up PR)